packages/@aws-cdk/aws-route53-targets/README.md

```diff
@@ -16,6 +16,13 @@ This library contains Route53 Alias Record targets for:
     // or - route53.RecordTarget.fromAlias(new alias.ApiGatewayDomain(domainName)),
   });
   ```
+* API Gateway V2 custom domains
+  ```ts
+  new route53.ARecord(this, 'AliasRecord', {
+    zone,
+    target: route53.RecordTarget.fromAlias(new alias.ApiGatewayv2Domain(domainName)),
```

Comment:
can the default domain of `ApiGatewayv2` be used as a target? (asking if we also need a class like `ApiGateway` for that scenario

Reply:
I also originally planned to provide something similar, but unlike the v1 version where you can just [provide a domain name string](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-apigateway-readme.html#custom-domains), v2 always requires domain name instances to be [created explicitly](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-apigatewayv2-readme.html#custom-domain).

This default domain name parameter is also then [setting the domain name of a default stage](https://github.com/aws/aws-cdk/blob/ddd29b485d5e07c0661ce135460b34d9f2b712e9/packages/%40aws-cdk/aws-apigatewayv2/lib/http/api.ts#L186-L190), which seemed fairly implicit as a target.

wdyt?

Reply:
gotcha, thanks for clarifying, I think this is fine as it is in that case.